go/vt/vtgate/engine/primitive.go

```diff
@@ -94,10 +96,8 @@ type Plan struct {
 	Rows uint64 `json:",omitempty"`
 	// Total number of errors
 	Errors uint64 `json:",omitempty"`
-	// NeedsLastInsertID signals whether this plan will need to be provided with last_insert_id
-	NeedsLastInsertID bool `json:"-"` // don't include in the json representation
-	// NeedsDatabaseName signals whether this plan will need to be provided with the database name
-	NeedsDatabaseName bool `json:"-"` // don't include in the json representation
+
+	sqlparser.BindVarNeeds
```

Comment:
nit: add comment for being in the plan like others. Also need to add json:"-" if this does not need json representation.